#BE-4073 Docs - Switching Organizations Update (#559)
appcircleio/appcircle-docusaurus: docs/account/my-organization/my-organization.md

You can switch between organizations at any time without any data loss.
:::

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4073-switch.png' />

To switch between organizations or sub organizations, follow these steps:

1. **Locate the Organization Switch Button:**

>   - At the bottom of your screen, find the button that displays the name of your current organization.

Reviewer: ```suggestion
  - Find the button displaying the name of your current organization at the bottom of your screen.
```

2. **Press the Organization Switch Button:**

>   - Click on this button to display a list of available organizations and sub-organizations that you are a part of.

Reviewer: ```suggestion
  - Click this button to display a list of available organizations and sub-organizations you are part of.
```

3. **Select the Desired Organization:**

>   - From the list, choose the organization or sub-organization you wish to switch to.

Reviewer: ```suggestion
  - From the list, choose the organization or sub-organization you want to switch to.
```
